lms/envs/common.py

```diff
@@ -773,7 +773,6 @@
 CMS_BASE = 'localhost:8001'
 
 # Site info
-SITE_ID = 1
```

Comment:
@mattdrayer It looks like this code is going to need a bit of refactoring. If the `SITE_ID` is not defined (which given that this was the only instance in settings, is never going to be), `get_current()` is going to throw a `ImproperlyConfigured` exception instead of `DoesNotExist` exception and code will not even go into this `except` block.

https://github.com/django/django/blob/1.8.9/django/contrib/sites/models.py#L50-L69